### PR TITLE
[USD] New port

### PR DESCRIPTION
**Pull request by cbezault**
Closes #3464

### CODE REVIEW

File: ports/usd/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO PixarAnimationStudios/USD
+    REF v19.05
+    SHA512 4d708835f6efd539d5fff5cbaf0ec4d68c6d0c4d813ee531c4b9589ee585b720c34e993ef0a7ad0104a921ebd7ab8dec46d0c9284ec7f11993057fe81d3729e0
+    HEAD_REF master)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    OPTIONS
+        -DBUILD_SHARED_LIBS:BOOL=${BUILDSHARED}
```
Comment (cenit) [inline ports/usd/portfile.cmake]: Not necessary ;)

File: ports/usd/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Don't file if the bin folder exists. We need exe and custom files.
+SET(VCPKG_POLICY_EMPTY_PACKAGE enabled)
+
+include(vcpkg_common_functions)
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
```
Comment (cenit) [inline ports/usd/portfile.cmake]: All this block looks unnecessary

File: ports/usd/portfile.cmake (hunk 1 above)
Comment (cbezault) [reply ports/usd/portfile.cmake]: Yeah, you're right, that anti-pattern made it in from the original PR (#3464). @vicroms is going to fix it.

**Pull request merged**